autogluon_zeroshot/simulation/config_generator.py

```diff
@@ -46,31 +48,88 @@ def select_zeroshot_configs(self,
             config_scorer = self.config_scorer
             selector = self._select_sequential
         while len(zeroshot_configs) < num_zeroshot:
-            iteration += 1
             # greedily search the config that would yield the lowest average rank if we were to evaluate it in combination
             # with previously chosen configs.
 
             valid_configs = [c for c in self.all_configs if c not in zeroshot_configs]
             if not valid_configs:
                 break
-
+            iteration += 1
 
             time_start = time.time()
-            best_next_config, best_score = selector(valid_configs, zeroshot_configs, config_scorer)
+            best_next_config, train_score_best = selector(valid_configs, zeroshot_configs, config_scorer)
```

Comment:
Nit comment: `best_train_score` is a bit better to keep the naming consistent with the first variable.

Reply:
updated